riscv/src/riscv.cpp

```diff
@@ -0,0 +1,54 @@
+#include "sim/memory.hpp"
```

Comment:
Please, use angle brackets in accordance with project codestyle

Reply:
fixed